crates/astria-sequencer-relayer/src/relayer/write/mod.rs

```diff
@@ -269,15 +297,56 @@ impl BlobSubmitter {
     }
 }
 
+/// Tries to confirm the last attempted submission of the previous session.
+///
+/// This should only be called where submission state on startup is `Prepared`, meaning we don't yet
+/// know whether that final submission attempt succeeded or not.
+///
+/// Internally, this polls `GetTx` for up to one minute.  The returned `SubmissionState` is
+/// guaranteed to be in `Finished` state, either holding the heights of the previously prepared
```

Comment:
Has the design of enums changed while writing this PR? `Finished` cleanly telegraphs what has happened, while the current return type `StartedSubmission` suggests that this method not only confirms the previous submission but also starts a new one.

Reply:
Yes, I originally had a `Finished` state too, but later removed it and forgot to update the comment.